vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -112,11 +112,18 @@ class FlyoutShadowNode : public ShadowNodeBase
   int64_t m_targetTag = -1;
   float m_horizontalOffset = 0;
   float m_verticalOffset = 0;
+  bool m_isFlyoutShowOptionsSupported = false;
   winrt::FlyoutShowOptions m_showOptions;
```

Comment:
Having m_showOptions which default initializes is still going to throw.  m_showOptions needs to be initialized as {nullptr} here

Reply:
Ok, I've updated the PR with a new commit.